RELEASING.md

```diff
@@ -67,47 +67,15 @@ To build Debian and RPM packages for a release:
 
 - Following the above, run `make deb rpm`.  You should see debian/changelog and
   rpm/felix.spec being updated with the new version number and release notes,
-  and packages built under `dist/`.  If you then publish those packages (for
-  which see below), you should also commit and push those changes, so that we
-  have a record of when and how the packages were built.
+  and packages built under `dist/`.
 
-To build Debian and RPM packages for any Git HEAD:
+- Create a PR to get those changes, in particular the release notes, reviewed.
+  If you need to make changes, do so, and run
 
-- Run `make deb rpm`.  You should see changes similar to the release case
-  above, except that you probably don't want to save the temporary
-  debian/changelog and rpm/felix.spec updates that are made.
+      FORCE_VERSION=<new version> make deb rpm
```

Comment:
Is `<new version>` the same as the above "new version" or does it need to be different?

Reply:
Yes, it should be the same as the `<new version>` in `make release VERSION=<new version>`.

Reply:
But that's a bit pants, isn't it?  It would perhaps be better and clearer to have something like `KEEP_CHANGELOGS=true make deb rpm`.

Reply:
That said, I think this PR should be just doc, and that if we want to make a change like that we should do it separately.